source/common/http/http2/codec_impl.h

```diff
@@ -175,6 +176,27 @@ class ConnectionImpl : public virtual Connection,
     ClientHttp2Options(const envoy::config::core::v3::Http2ProtocolOptions& http2_options);
   };
 
+  class BufferMemoryAccount : public Account,
+                              public std::enable_shared_from_this<BufferMemoryAccount> {
+  public:
+    BufferMemoryAccount() = default;
+    ~BufferMemoryAccount() = default;
```

Comment:
nit: `~BufferMemoryAccount() = default;` does not need to be added explicitly.

Technically this should be `~BufferMemoryAccount() override`, but removing it is better.